packages/core/src/cli/perform-check.ts

```diff
@@ -1,8 +1,10 @@
-import type ts from 'typescript';
+import ts from 'typescript';
```

Comment:
```suggestion
import type ts from 'typescript';
```

Reply:
4bac7479c (also updated to use `TS` instead of `ts` here)